### PR TITLE
Fix _unpack_addr when using a unicode spec in Python2

### PR DESCRIPTION
**Pull request by anler**
Hi! I'm trying to use the library with Python2 and server weights and I'm having issues due to the str instance check [`_unpack_addr`](https://github.com/lericson/pylibmc/blob/1.6.1/src/pylibmc/client.py#L39).

I'm proposing this small change to take into account spec strings which are unicode.

### CODE REVIEW

**Review by anler — COMMENTED** [review]
Hi! I'm trying to use the library with Python2 and server weights and I'm having issues due to the str instance check _unpack_addr. I'm proposing this small change to take into account spec strings which are unicode.

File: src/pylibmc/client.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def _unpack_addr(spec, port=None, weight=1):
     else:
         addr = spec
 
-    if isinstance(port, str) and ":" in port:
+    if isinstance(port, (str, unicode)) and ":" in port:
```
Comment (lericson) [inline src/pylibmc/client.py:39]: This will not work in Python 3.x, and Python 2.x has been removed as a target platform.

**Comment by lericson** [comment]
If you would like to start maintaining a Py 2.x variant of pylibmc, I'd suggest starting from commit 5b1dbd3 which is the last commit before I merged the Py2k drop PR from @hugovk.

**Comment by lericson** [comment]
It does look like this PR should be added on top of that though since _unpack_addr never did actually work for unicode strings in Py2k.

**Pull request closed without merging**